src/openforms/appointments/tests/test_base.py

```diff
@@ -45,3 +51,23 @@ def test_get_appointment_details_html(self):
         self.assertIn("Remarks", result)
         self.assertIn("Some", result)
         self.assertIn("<h1>Data</h1>", result)
+
+    @patch("openforms.appointments.base.submission_appointment_token_generator")
+    def test_get_appointment_links(self, token_generator_mock):
+        submission = SubmissionFactory.create()
+        fake_token = "fake-token"
+
+        token_generator_mock.make_token.return_value = fake_token
+
+        result = self.plugin.get_appointment_links(submission)
+
+        cancel_path = reverse(
+            "appointments:appointments-verify-cancel-appointment-link",
+            kwargs={
+                "token": fake_token,
+                "submission_uuid": submission.uuid,
+            },
+        )
+        cancel_url = urljoin(settings.BASE_URL, cancel_path)
```

Comment:
use `@override_settings(BASE_URL=...)` instead and verify that the link is indeed the right fully qualified URL.